tests/unit/test_config.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import asdict
+
+import pytest
+
+from sutta_publisher.shared.config import Config
+
+
+@pytest.mark.vcr
+def test_config_for_publication(scpub1_data):
+    pub_config = Config.from_publication("scpub1")
+
+    assert asdict(pub_config) == scpub1_data
```

Comment:
Nice! and I'd go with explicit checks. `scpub1_data` - it really can bite us when it starts to depend on other tests and json from the repo will change, hence:
```suggestion
    assert pub_config.creator_name == "Bhikkhu Sujato"
    assert pub_config.translation_title == "Middle Discourses"
    ...
```
Those are tests so we want them to be verbose and simple